pkgs/tools/archivers/arc_unpacker/install.cmake

```diff
@@ -0,0 +1,15 @@
+include(GNUInstallDirs)
```

Comment:
Should probably either upstream this, or simply use a custom `installPhase`.

Reply:
I went with a custom `installPhase` for now.